### DISCUSSION TITLE
关于是否能够支持展示 OneDrive 根目录的疑问

### DISCUSSION THREAD

**Discussion by ghost** [spencerwooo/onedrive-vercel-index#157] [Q&A] [answered]
我正试图将 OneDrive 根目录载入页面，为此，我尝试了将 `config/api.json` 中的 `base` 条目修改为  `/`、`\` 、`*` 或者设置为空，均以 500 错误告终。关于是否能够展示根目录，我也没能找到确切的描述。所以，我想知道这个项目支持展示根目录吗？😥

**Comment by myl7** [comment, marked as answer]
参见 #159 ，我本地试了一下没问题了

**Reply by ghost** [reply]
非常感谢~
